tests/dream/geant4_reduction_test.py

```diff
@@ -57,8 +58,8 @@
     WavelengthMask: None,
     CIFAuthors: CIFAuthors(
         [
-            CIFAuthor(
-                name="Jane Doe", email="jane.doe@ess.eu", id_orcid="0000-0000-0000-0001"
+            Author(
```

Comment:
Would it be better to add one more `Author` for testing since it's `CIFAuthors`...?